int64 id
ltlplanner/goal goal
